xscen/diagnostics.py

```diff
@@ -195,30 +195,30 @@ def properties_and_measures(
     if unstack:
         ds = unstack_fill_nan(ds)
 
-    unit_conversion = unit_conversion or {}
-    for var, unit in unit_conversion.items():
-        ds[var] = convert_units_to(ds[var], unit)
+    if change_units_arg:
+        ds = change_units(ds, variables_and_units=change_units_arg)
 
     prop = xr.Dataset()  # dataset with all properties
     meas = xr.Dataset()  # dataset with all measures
     for i, ind in enumerate(properties, 1):
         if isinstance(ind, tuple):
             iden, ind = ind
-        else:
-            iden = ind.identifier
-        logger.info(f"{i} - Computing {iden}.")
-
         # Make the call to xclim
-        prop[iden] = ind(ds=ds)
+        out = ind(ds=ds)
+        vname = out.name
+        logger.info(f"{i} - Computing {vname}.")
```

Comment:
Je comprends l'idée, mais je trouve ça potentiellement mêlant si l'appel de l'indicateur bogue. Il serait mieux que le log indique on est rendu où _avant_.
```suggestion
        logger.info(f"{i} - Computing {iden}.")
        out = ind(ds=ds)
        vname = out.name
```
Une autre option serait `ind.cf_attrs[0]['var_name']`, où on aurait le nom de variable, potentiellement avec les balises non formattées.

Reply:
ah oui en effet